pkg/rancher-components/src/components/LabeledTooltip/LabeledTooltip.vue

```diff
@@ -29,9 +29,14 @@ export default Vue.extend({
     }
   },
   computed: {
-    iconClass() {
+    iconClass(): string {
       return this.status === 'error' ? 'icon-warning' : 'icon-info';
     }
+  },
+  methods: {
+    isObject(value: string | Record<string, unknown>): value is Record<string, unknown> {
+      return typeof value === 'object' && value !== null && !!value.content;
+    }
```

Comment:
Could lodash be used here? I'm not sure what the dependencies look like for rancher-components

Reply:
We don't import the entirety of lodash inside of `rancher-components`, so we can add an import for `lodash.isobject`[^1] to support this. 

[^1]: https://www.npmjs.com/package/lodash.isobject